--- photostory	2010-08-16 17:23:41.744783667 +0100
+++ photostory	2010-08-16 17:29:36.963710900 +0100
@@ -211,7 +211,7 @@
         win.set_default_size(900, 600)
         win.set_resizable(False)
         win.set_title("Photostory")
-        win.set_icon_from_file("data/photostory.svg")
+        win.set_icon_from_file("/usr/local/share/photostory/data/photostory.svg")
         movie = gtk.DrawingArea()
         hbox = gtk.HBox(homogeneous=False, spacing=3)
         vbox1 = gtk.VBox(homogeneous=False)
